src/components/Reports/Shared/CalculationReports/DirectionButtons/DirectionButtons.tsx

```diff
@@ -52,31 +67,34 @@ export const DirectionButtons: React.FC<DirectionButtonsProps> = ({
             <ChevronLeft sx={{ mr: 1 }} />
             {t('Back')}
           </Button>
+        )}
+        {isSubmission ? (
           <Button
             variant="contained"
             color="primary"
             onClick={handleConfirm}
-            disabled={submitCount > 0 && !isValid}
+            disabled={submitCount ? submitCount > 0 && !isValid : false}
           >
             {t('Submit')}
             <ChevronRight sx={{ ml: 1 }} />
           </Button>
-          {openConfirmation && (
-            <SubmitModal
-              handleClose={() => setOpenConfirmation(false)}
-              handleConfirm={submitForm}
-            />
-          )}
-        </Box>
-      ) : (
-        <Button
-          variant="contained"
-          color="primary"
-          onClick={handleNext ?? handleNextStep}
-        >
-          {t('Continue')}
-          <ChevronRight sx={{ ml: 1 }} />
-        </Button>
+        ) : (
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={overrideNext ?? handleNextStep}
+          >
+            {buttonTitle ?? t('Continue')}
+            <ChevronRight sx={{ ml: 1 }} />
+          </Button>
+        )}
+      </Box>
+      {openConfirmation && (
+        <SubmitModal
+          formTitle={t('MHA Request')}
+          handleClose={() => setOpenConfirmation(false)}
+          handleConfirm={submitForm ? submitForm : async () => {}}
```

Comment:
Do you need to make the empty function async? Also I guess you also need to close the modal unless you do that already on the handleConfirm in the submitModal